contracts/HolographOperator.sol

```diff
@@ -186,20 +183,23 @@ contract HolographOperator is Admin, Initializable, IHolographOperator {
   mapping(address => uint256) private _bondedOperators;
 
   /**
-   * @dev Internal mapping of bonded operator amounts.
+   * @dev Internal mapping of bonded operators, to prevent double bonding.
    */
-  mapping(address => uint256) private _bondedAmounts;
+  mapping(address => uint256) private _operatorPodIndex;
 
   /**
-   * @dev Event is emitted for every time that a valid job is available.
+   * @dev Internal mapping of bonded operator amounts.
    */
-  event AvailableOperatorJob(bytes32 jobHash, bytes payload);
+  mapping(address => uint256) private _bondedAmounts;
 
   modifier onlyBridge() {
     require(msg.sender == _bridge(), "HOLOGRAPH: bridge only call");
     _;
   }
 
+  /**
+   * @dev Allow calls only from LayerZero endpoint contract.
+   */
   modifier onlyLZ() {
     assembly {
       // check if lzEndpoint
```

Comment:
It could be nice to have the hashes in the `mstore` be constants or add comments to some of these assembly blocks that are doing more than setting/getting a variable. OpenSea's SeaPort codebase does a really nice job of this: https://github.com/ProjectOpenSea/seaport/blob/171f2cd7faf13b2bf0455851499f1981274977f7/contracts/lib/GettersAndDerivers.sol

Reply:
I will provide descriptions and explanations inside of assembly functions to explain what is happening. This is gonna happen in the next iteration when I'll be commenting and documenting all the functions.

Reply:
Nice sounds good